### PR TITLE
Add `clipboard-plus` and `clipboard-minus` icons

### PR DESCRIPTION
**Pull request by ericfennis**
## What is the purpose of this pull request?

- [x] New Icon
- [ ] Bug fix
- [ ] New Feature
- [ ] Documentation update
- [ ] Other:

### Description
Adding `clipboard-plus` and `clipboard-minus`.

### Icon use case 
- Adding something to the clipboard
- `clipboard-plus` could be used for medical use cases (not ideal), like doctor reports.
- Removing something from the clipboard

### Concept 

- [x] I have provided valid use cases for each icon.
- [x] I have not added any a brand or logo icon.
- [x] I have not used any hate symbols.
- [x] I have not included any religious or political imagery.

### Author, credits & license

- [x] The icons are solely my own creation.
- [x] I've based them on the following Lucide icons: 
- [x] I've based them on the following design: 

### Naming 

- [x] I've read and followed the [naming conventions](https://lucide.dev/guide/design/icon-design-guide#naming-conventions)
- [x] I've named icons by what they are rather than their use case.
- [x] I've provided meta JSON files in `icons/[iconName].json`.

### Design 

- [x] I've read and followed the [icon design guidelines](https://lucide.dev/guide/design/icon-design-guide)
- [x] I've made sure that the icons look sharp on low DPI displays.
- [x] I've made sure that the icons look consistent with the icon set in size, optical volume and density.
- [x] I've made sure that the icons are visually centered.
- [x] I've correctly optimized all icons to two points of precision.

## Before Submitting 

- [x] I've read the [Contribution Guidelines](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md).
- [x] I've checked if there was an existing PR that solves the same issue.

### CODE REVIEW

**Comment by jguddas** [comment]
Do we have a standard plus and minus size, feels a bit big.

**Comment by ericfennis** [comment]
@jguddas I've based these ones on the `plus-square` icons.

**Comment by ericfennis** [comment]
The plus from `file-plus` is also an option.
image

100%
plus

File: icons/clipboard-minus.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "ericfennis"
```
Comment (jguddas) [inline icons/clipboard-minus.json:4]: Should include the contributors from the icon you based this on IMO.

**Review by jguddas — APPROVED** [review]

**Comment by karsa-mistmere** [comment]
I agree with @jguddas, the plus and minus signs feel a bit too large, almost all of our other `*-plus` icons use a 6 by 6 plus (other than a few outliers such as `square-plus` and `circle-plus` which have plenty of space to accommodate for larger signs, and `calendar-plus` that uses a tiny one):
image

File: icons/shield-plus.json
Comment (karsa-mistmere) [inline icons/shield-plus.json:1]: Maybe we should also decrease the plus size on `shield-plus` to match? 🤔
image

Comment (ericfennis) [reply icons/shield-plus.json:1]: Yess, will do in a separate PR!

**Review by karsa-mistmere — APPROVED** [review]

**Review by jguddas — APPROVED** [review]

**Pull request merged**